functions/colab.js

```diff
@@ -48,7 +48,10 @@ exports.handler = async function (event, _) {
     '!pip install -r requirements.txt'
   ]
 
-  const execution_nb_commands = ['!python main.py config.yaml']
+  const argparser = data.argparser
+  const execution_nb_commands = [
+    `!python main.py ${argparser === 'hydra' ? '' : 'config.yaml'}`
```

Comment:
provide here explicit instruction that defines the config.yaml as arg to show how to change the config file.